### ISSUE TITLE
folder_contents Configure display column dialog half visible

### ISSUE THREAD

**Issue by vincentfretin** [plone/Products.CMFPlone#3124]
### What I did:

Create a Plone site in German
Go in folder contents, click on "Configure display column"

### What I expect to happen:
The dialog should be completely visible.

### What actually happened:

The dialog is half visible, some part is below the toolbar
![Capture d’écran de 2020-06-19 11-58-36](https://user-images.githubusercontent.com/112249/85120941-3ec6bc80-b224-11ea-92cf-cbb9f5f1bafa.png)

This is what it looks like in English:
![Capture d’écran de 2020-06-19 12-01-20](https://user-images.githubusercontent.com/112249/85121197-a7ae3480-b224-11ea-8901-cc90b4936641.png)

### What version of Plone/ Addons I am using:
Plone 5.2.1

**Comment by vincentfretin** [comment]
When I put plone-editor-tools bundle in develop mode, the issue is fixed!
So the issue was surely fixed by someone, but the bundle wasn't rebuild?

**Comment by vincentfretin** [comment]
The question now is how to build this bundle to include it in a new release? I don't have a clue. Someone knows?

**Comment by vincentfretin** [comment]
To create the new bundles, the documentation is at https://github.com/plone/plone.staticresources

**Comment by vincentfretin** [comment]
The bundles have been updated recently by @thet last_compilation 2020-06-16 10:10:00
and I see in plone.staticresources package.json with the latest commit of mockup master https://github.com/plone/mockup.git#e6bd1977871c7dbeb75b0eccc395687320e681b8

Huh, now I played a little bit with plone.staticresources it appears this is not an issue with released vs develop mode, weird. I can reproduce the issue in develop mode. The issue is not existing in plone.staticresources 1.2.1, it only occurs on master.

We have those CSS rules:
```
http://0.0.0.0:8080/Plone/++plone++static/++unique++2020-06-21%2010%3A20%3A26.676322/src/plone.staticresources/src/plone/staticresources/static/components/bootstrap/less/popovers.less
.pat-structure .popover {
 z-index: 1060;
}

http://0.0.0.0:8080/Plone/++plone++static/++unique++2020-06-21%2010%3A20%3A26.676322/src/mockup/mockup/patterns/structure/less/pattern.structure.less
.pat-structure .navbar, .pat-structure .pat-structure .navbar {
 z-index: 10;
}
```
toolbar is z-index: 20

This is what changed, in the DOM, now we have:
```
<div class="navbar ui-offset-parent>
  <div id="popover-structure-columns" class="popover attribute-columns ui-offset-parent active bottom">
  </div>
</div>
```

And before we had
```
<div class="navbar ui-offset-parent>
</div>
<div id="popover-structure-columns" class="popover attribute-columns ui-offset-parent active bottom"></div>
```

**Comment by vincentfretin** [comment]
![Capture d’écran de 2020-06-21 10-31-43](https://user-images.githubusercontent.com/112249/85220477-12d64300-b3ac-11ea-8a39-c37cd21ffda3.png)
![Uploading Capture d’écran de 2020-06-21 10-32-57.png…]()

**Comment by vincentfretin** [comment]
I couldn't find what changed in the js to have a different DOM like this.
The issue can be fixed by changing the CSS rule for `.pat-structure .navbar` from z-index: 10 to z-index: 1060. I'm not sure if this can lead to other regressions somewhere else.

**Comment by vincentfretin** [comment]
I did the css fix there https://github.com/plone/mockup/pull/986/commits/a6fa639b9e80ec04d9143bf451e73fd51d6ff0ae
waiting merge of https://github.com/plone/mockup/pull/986

**Comment by vincentfretin** [comment]
This issue was a regression because of https://github.com/plone/mockup/pull/964
The z-index of children is only applied within the same stacking context, see 
https://coder-coder.com/z-index-isnt-working/
for more details.

**Comment by vincentfretin** [comment]
The fix I did included in mockup 3.2.0 did regressions with submenu in Plone toolbar and portal tabs subtrees simlar to https://github.com/plone/Products.CMFPlone/issues/2307 that was actually fixed before.
My new PR https://github.com/plone/mockup/pull/1002 revert the previous fix and fix this issue in a different way.
